### DISCUSSION TITLE
How do I use multiple characters as field separators?

### DISCUSSION THREAD

**Discussion by easylinked** [osiegmar/FastCSV#77] [Q&A] [answered]
In the source code, CsvReader and CsvWriter, the CsvWriterBuilder class fieldSeparator field is of type char. How can I use Tab and custom multi-characters as field separators?
![image](https://user-images.githubusercontent.com/126239592/224243181-0034dcf2-8dbf-4ce5-8b96-58117f1e3e29.png)

**Comment by osiegmar** [comment, marked as answer]
A Tab character can be easily configured by `CsvReader.builder().fieldSeparator('\t')`. Multi-Character values (for fieldSeparator, quoteCharacter and commentCharacter) are not possible.

**Comment by easylinked** [comment]
Thx

**Comment by imerljak** [comment]
Why is this exact feature listed in the README if it isn't supported?
It is clearly stating that delimiters can be mixed, and even auto-detected, which also isn't the case.

![Screenshot 2023-10-02 at 13 21 38](https://github.com/osiegmar/FastCSV/assets/17605203/c490aa53-404d-4a39-85f3-b81e4422e9bb)

**Reply by osiegmar** [reply]
Don't mix up **line** and **field** delimiters. **Line delimiters** (CR, LF, CRLF) are auto-detected – even if mixed. **Field delimiters** have to be configured (`,` by default) and cannot be mixed.
